### PR TITLE
Merchant items

### PR DESCRIPTION
**Pull request by Alex-Desjardins**

### CODE REVIEW

File: app/models/item.rb [hunk 1]
```diff
@@ -6,9 +6,9 @@ class Item <ApplicationRecord
 
   validates_presence_of :name,
                         :description,
-                        :price,
-                        :image,
-                        :inventory
+                        :price
+  validates_presence_of :image, allow_blank: true
```
Comment (dcoleman21) [inline app/models/item.rb:10]: Unsure how to use placeholder image

File: app/views/merchant/items/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  <%= text_field_tag :image, (if @item.present? then @item.image end) %>
+
+  <%= label_tag :price %>
+  <%= text_field_tag :price, (if @item.present? then @item.price else nil end) %>
+
+  <%= label_tag :inventory %>
+  <%= text_field_tag :inventory, (if @item.present? then @item.inventory else nil end) %>
+
+  <%= submit_tag "Update Item" %>
+<% end %>
```
Comment (hashmaster3k) [inline app/views/merchant/items/edit.html.erb:18]: Good spot to use partials and form_for

File: spec/models/item_spec.rb [hunk 3]
```diff
@@ -5,8 +5,8 @@
     it { should validate_presence_of :name }
     it { should validate_presence_of :description }
     it { should validate_presence_of :price }
-    it { should validate_presence_of :image }
-    it { should validate_presence_of :inventory }
+    #it { should validate_presence_of :image }
+    it { should validate_numericality_of :inventory }
```
Comment (dcoleman21) [inline spec/models/item_spec.rb:9]: wow that is a neat word

**Review by dcoleman21 — APPROVED** [review]
Looks so good team

File: spec/features/merchant_users/items/edit_spec.rb [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+    ItemOrder.create(item_id: @paper.id , order_id: @order_1.id, price: @paper.price, quantity: 1)
+
+    visit '/login'
+
+    fill_in :email, with: @merchant.email
+    fill_in :password, with: @merchant.password
+
+    click_button 'Log in'
+
+    visit "/merchant"
```
Comment (hashmaster3k) [inline spec/features/merchant_users/items/edit_spec.rb:40]: Could save some lines by using allow_any_instance_of

**Pull request merged**